validation/src/test/scala/hmda/validation/rules/lar/validity/RaceEditCheckSpec.scala

```diff
@@ -0,0 +1,57 @@
+package hmda.validation.rules.lar.validity
+
+import hmda.model.fi.lar.{ Applicant, LoanApplicationRegister }
+import hmda.validation.rules.lar.LarEditCheckSpec
+
+abstract class RaceEditCheckSpec extends LarEditCheckSpec {
+
+  def succeedsWithRace(vals: List[String]) = {
+    forAll(larGen) { lar: LoanApplicationRegister =>
+      val testApplicant = applicantWithRaceVals(lar.applicant, vals)
+      val testLar = lar.copy(applicant = testApplicant)
+      testLar.mustPass
+    }
+  }
+  def succeedsWithCoRace(vals: List[String]) = {
+    forAll(larGen) { lar: LoanApplicationRegister =>
+      val testApplicant = applicantWithCoRaceVals(lar.applicant, vals)
+      val testLar = lar.copy(applicant = testApplicant)
+      testLar.mustPass
+    }
+  }
+
+  def failsWithRace(vals: List[String]) = {
+    forAll(larGen) { lar: LoanApplicationRegister =>
+      val testApplicant = applicantWithRaceVals(lar.applicant, vals)
+      val testLar = lar.copy(applicant = testApplicant)
+      testLar.mustFail
+    }
+  }
+  def failsWithCoRace(vals: List[String]) = {
```

Comment:
I very much like having these methods. Can you say more about why they take a List rather than 5 args? I can see that it makes it easier to pass the chunk of vals down into the next method, and maybe 5 different names wouldn't be meaningful, but I also notice that it does mean you're writing `List` each time, and that it means the compiler can't yell if you pass the wrong number of values. I'm not sure how to weigh these things, and I suspect you have reasons that I haven't considered (and have spent more time thinking about those I have), so I'd like to hear what you say.